.github/workflows/test.yml

```diff
@@ -5,19 +5,19 @@ on: [push, pull_request]
 jobs:
   build:
     name: build (Python ${{ matrix.python-version }}, Django ${{ matrix.django-version }})
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-18.04
```

Comment:
This is basically to work around an issue with installing mysqlclient on Ubuntu 20.04 that I couldn't fix:

```
ImportError: libmysqlclient.so.20: cannot open shared object file: No such file or directory
```

Reply:
Hmm, I can look into this later.